Updated db schema path (#4240)
magento/devdocs: guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

</tr>
</table>

> For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` directory.

Reviewer: Hi @sanganinamrata. Please, add also:

"or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`". 

There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.

Author: @rogyar Sure, I'll do it and let you know.

Author: @rogyar I have updated.

</tr>
</table>

> For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/setup/src/Magento/Setup/Model/Declaration/Schema/etc` or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc` directory. There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.

Reviewer 2: Please correlate each path with a specific installation method.

Author: @jeff-matthews I have updated, Can you please proceed this PR?

> For more information about each type, refer to the annotations in the corresponding XSD file. The XSD files are located in the `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` (If you have downloaded Magento from here https://magento.com/tech-resources/download) or `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc` directory (If you have installed via Github https://github.com/magento/magento2 or using composer). There are two possible places where the mentioned XSD files can be located. It depends on how Magento is installed.

Reviewer 2: ```suggestion
For more information about each type, refer to the annotations in the corresponding XSD file. The location of the XSD file depends on how you installed Magento.

- [Archive download]({{page.baseurl}}/install-gde/prereq/zip_install.html): `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc` 
- [Composer]({{page.baseurl}}/install-gde/composer.html) or [GitHub]({{page.baseurtl}}/install-gde/prereq/dev_install.html) installation: `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`
```
